src/binder_netreg.c

```diff
@@ -1451,6 +1712,79 @@ binder_netreg_get_signal_strength_dbm(
     }
 }
 
+static
+int
+binder_netreg_get_signal_strength_dbm_aidl(
+    GBinderReader* reader)
+{
+    int rssi = -1, rscp = -1, rsrp = -1;
+    const RadioSignalStrengthGsm* gsm;
+    const RadioSignalStrengthLte* lte;
+    const RadioSignalStrengthTdScdma_1_2* tdscdma;
+    const RadioSignalStrengthWcdma_1_2* wcdma;
+    const RadioSignalStrengthNr* nr;
+
+    binder_read_parcelable_size(reader);
+
+    gsm = gbinder_reader_read_parcelable(reader, NULL);
+    gbinder_reader_read_parcelable(reader, NULL); /* cdma */
+    gbinder_reader_read_parcelable(reader, NULL); /* evdo */
+    lte = gbinder_reader_read_parcelable(reader, NULL);
+    tdscdma = gbinder_reader_read_parcelable(reader, NULL);
+    wcdma = gbinder_reader_read_parcelable(reader, NULL);
+    nr = gbinder_reader_read_parcelable(reader, NULL);
+
+    if (gsm->signalStrength <= RSSI_MAX) {
+        rssi = gsm->signalStrength;
+    }
+
+    if (lte->signalStrength <= RSSI_MAX &&
+        (int)lte->signalStrength > rssi) {
```

Comment:
Nitpick: this style of lack of indentation makes this quite difficult to read.

Reply:
This is just using same formatting as the existing binder_netreg_get_signal_strength_dbm function. I agree the formatting is not the best.

Reply:
Yes but after looking the changes different styles are used even within one source file...